config/webpack.config.production.js

```diff
@@ -47,7 +70,55 @@ module.exports = merge(baseConfig, {
     new webpack.DefinePlugin({
       'process.env.NODE_ENV': JSON.stringify('production')
     }),
-    new BabiliPlugin(),
+    new UglifyJSPlugin({
```

Comment:
Does uglifyJS support ES6 which we are using? I think the reason I used Babili was that it was the only one that supported ES6 out of the box.

Reply:
Jep. Documentation says, that i can use the `ecma` option (https://github.com/webpack-contrib/uglifyjs-webpack-plugin#uglifyoptions)